src/crimson/osd/pg.cc

```diff
@@ -610,6 +610,25 @@ osd_op_params_t&& PG::fill_op_params_bump_pg_version(
     osd_op_p.user_at_version = osd_op_p.at_version.version;
   }
   return std::move(osd_op_p);
+
+seastar::future<Ref<MOSDOpReply>> PG::do_failed_op_processing(
```

Comment:
how about s/do_failed_op_processing/reply_failed_op/ or s/do_failed_op_processing/handle_failed_op/ ?

Reply:
`reply_failed_op` would be too-focused and *replying* while we have even more important responsibility there. However, `handle_failed_op` is good! Taking it, thanks!